lib/generators/pakyow/app/templates/app/setup.rb

```diff
@@ -13,6 +13,7 @@
 
     # put global config here and they'll be available across environments
     app.name = '<%= app_name %>'
+    logger.enabled = true
```

Comment:
Definitely!  It's because the logger.enabled attribute doesn't exist in older Pakyow versions but is referenced within the template itself.  So when a new Pakyow app is generated with the master version of Pakyow the template has a conditional that references logger.enabled (in templates/middleware/logger.rb).  However that template's Gemfile references an older version of Pakyow so the logger.enabled config value doesn't exist, which causes an error.  As opposed to focusing on the approach of locking down the Pakyow template version to be the same as the version that generated it, which to me seems less desirable, declaring that logger.enabled value in the setup config file allows older versions to have a reference to it.

A better approach actually might be to just alter the conditional in middleware/logger.rb to accommodate both config options.

What are your thoughts?